docs/source/api/inference.rst

```diff
@@ -55,6 +55,13 @@ ADVI
 
 .. currentmodule:: pymc3.variational.advi
 
+.. autosummary::
+
+   TestFunction
```

Comment:
They aren't in `pymc3.variational.advi`, but in `pymc3.variational.inference`, so this doesn't work.